e2e/async-esm-import/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "async-esm-import",
+  "type": "module",
+  "jest": {
+    "transform": {},
+    "testEnvironment": "node",
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(js?|ts?|mjs?|mts?)$"
```

Comment:
Thanks. All is good for my eye.

Only this line looks redundant. There aren’t any `mjs` or `mts` tests anymore.

```suggestion
```

Reply:
Ohhh, I was careless again. Done, and thanks for your patience.